src/jdk.httpserver/share/classes/sun/net/httpserver/simpleserver/resources/simpleserver.properties

```diff
@@ -23,10 +23,17 @@
 # questions.
 #
 
-usage=\
+usage.java=\
 Usage: java -m jdk.httpserver [-b bind address] [-p port] [-d directory]\n\
 \                              [-o none|info|verbose] [-h to show options]
 
+usage.jwebserver=\
```

Comment:
should usage also show `-version, --version` in both usage strings, and in the usage summary of the man page below as well?